hax/treemakers/corrections.py

```diff
@@ -53,6 +53,14 @@ class Corrections(TreeMaker):
 
     extra_metadata = hax.config['corrections_definitions']
 
+    #Check if data was generated with MC.
+    #If it was, pull the electron lifetime values from the associated metadata
+    def check_for_mc(self):
+        mc_data = False
+        if 'MC' in hax.paxroot.get_metadata(self.run_name)['configuration']:
```

Comment:
Can this function be moved to ```runs.py```? (Looks like maybe some duplicated code there.)

Reply:
Maybe. The function in runs.py, checks if it is MC data as well, but is only used to set a run number, whereas this function is used to make the decision between using the interpolated lifetime or the correction init file electron lifetime